Composer/packages/client/src/pages/botProject/SkillHostEndPoint.tsx

```diff
@@ -70,7 +70,14 @@ export const SkillHostEndPoint: React.FC<SkillHostEndPointProps> = (props) => {
   const { skillHostEndpoint } = useRecoilValue(settingsState(projectId));
 
   return (
-    <CollapsableWrapper title={formatMessage('Skill host endpoint')} titleStyle={title}>
+    <Fragment>
+      <div css={title}>{formatMessage('Call skills')}</div>
```

Comment:
Is 'Call skills' correct? I thought this was supposed to be settings to allow the bot to be used as a skill.

Reply:
I am running this by final verification with design today, Will bring this up.

Reply:
Text will stay as is. There is an all up task to update copy across the settings and provisioning pages which will be handled in its own PR.